### DISCUSSION TITLE
Changing decimal separator from number input

### DISCUSSION THREAD

**Discussion by jaqueamo** [cefsharp/CefSharp#4159] [General]
Hello, 

having this code loaded into the browser:

`<input type="number" value="1.99" />`

I get the follwoing output:
![output](https://user-images.githubusercontent.com/7593386/178425332-56cd035f-457f-4364-8468-3fa53ca3fbd0.png)

What I want to achieve is to change the decimal separator from comma to dot.

What I already tried:

Set the language in the CefSettings:
```
settings.Locale = "en-US";
settings.AcceptLanguageList = "en-US";
```

Changed the langauge of the browser:
```
ChromiumWebBrowser.Language = System.Windows.Markup.XmlLanguage.GetLanguage("en-US");
```

Changed the Culture of the thread where ChromiumWebBrowser is instantiated:
```
Thread.CurrentThread.CurrentCulture = System.Globalization.CultureInfo.GetCultureInfo("en-US");
Thread.CurrentThread.CurrentUICulture = System.Globalization.CultureInfo.GetCultureInfo("en-US");
```

Added language attribute to the html page:
```
<html lang="en">
```

Added language attribute to the input:
```
<input type="number" value="1.99" lang="en"/>
```

But the decimal separator is still comma.

Any advice what I can do? 

Thank you :-)

**Comment by amaitland** [comment]
What version are you using? 

How does the same version of Chrome behave?

**Comment by jaqueamo** [comment]
I'm using CefSharp.Wpf version 103.0.90
Chrome (103.0.5060.114) has the same behaviour - also uses comma instead of dot. 

My region is set to Germany in the Windows settings. If I try it in a virtual machine running Windows set to English (US) region the dot is displayed (for Chrome and CefSharp).

Isn't there a way to override these settings for CefSharp?

**Reply by amaitland** [reply]
At it's core you are using chromium so behaviour matching chrome is generally expected. 

You can try a different locale like en-GB (making sure you've set a CachePath).

> Isn't there a way to override these settings for CefSharp?

If that doesn't work then you'll have to move the conversation to https://magpcss.org/ceforum/index.php as CEF provides the actual chromium integration.
